docs/docs/Flows/lfx.mdx

```diff
@@ -0,0 +1,312 @@
+
+    ```
+    uv run lfx serve simple-agent-flow.json
+    ```
+
+    To export new values, stop the server, export the variables, and then start the server again.
+
+    </TabItem>
+    </Tabs>
+3. The startup process displays a `flow_id` value in the output.
+   Copy the `flow_id` to use in the test API call in the next step.
+   In this example, the `flow_id` is `c1dab29d-3364-58ef-8fef-99311d32ee42`.
+
+    ```bash
+     ╭───────────────────────────── LFX Server ─────────────────────────────╮
+     │ 🎯 Single Flow Served Successfully!                                  │
+     │                                                                      │
+     │ Source: /Users/mendonkissling/Downloads/simple-agent-flow.json       │
+     │ Server: http://127.0.0.1:8000                                        │
+     │ API Key: sk-...                                                 │
+     │                                                                      │
+     │ Send POST requests to:                                               │
+     │ http://127.0.0.1:8000/flows/c1dab29d-3364-58ef-8fef-99311d32ee42/run │
+     │                                                                      │
+     │ With headers:                                                        │
+     │ x-api-key: sk-...                                               │
+     │                                                                      │
+     │ Or query parameter:                                                  │
+     │ ?x-api-key=sk-...                                               │
+     │                                                                      │
+     │ Request body:                                                        │
+     │ {'input_value': 'Your input message'}                                │
+     ╰──────────────────────────────────────────────────────────────────────╯
+    ```
+
+4. In a new terminal, export your `flow_id` and Langflow API key values as variables:
+    ```bash
+    export LANGFLOW_API_KEY="sk..."
+    export FLOW_ID="c1dab29d-3364-58ef-8fef-99311d32ee42"
+    ```
+
+5. Test the server with an API call to the `/flows/flow_id/run` endpoint:
+
+    ```bash
+    curl -X POST http://localhost:8000/flows/$FLOW_ID/run \
+      -H "Content-Type: application/json" \
+      -H "x-api-key: $LANGFLOW_API_KEY" \
+      -d '{"input_value": "Hello, world!"}'
+    ```
+
+    Successful response example:
+    ```json
+    {
+      "result": "Hello world! 👋\n\nHow can I help you today? If you have any questions or need assistance, just let me know!",
+      "success": true,
+      "logs": "\n\n\u001b[1m> Entering new None chain...\u001b[0m\n\u001b[32;1m\u001b[1;3mHello world! 👋\n\nHow can I help you today? If you have any questions or need assistance, just let me know!\u001b[0m\n\n\u001b[1m> Finished chain.\u001b[0m\n",
+      "type": "message",
+      "component": "Chat Output"
+    }
+    ```
```

Comment:
Include the structure of the returned output, because it's different the from Langflow API